tests/blockchain/test_blockchain.py

```diff
@@ -2074,7 +2074,7 @@ async def test_aggsig_garbage(self, empty_blockchain, opcode, with_garbage, expe
         synthetic_secret_key = calculate_synthetic_secret_key(secret_key, DEFAULT_HIDDEN_PUZZLE_HASH)
         public_key = synthetic_secret_key.get_g1()
 
-        args = [public_key, b"msg"] + ([b"garbage"] if with_garbage else [])
+        args = [bytes(public_key), b"msg"] + ([b"garbage"] if with_garbage else [])
```

Comment:
args is `List[bytes]`

Reply:
Did mypy catch this?  Or should be explicitly `: List[bytes]` to make sure it is enforced.